cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -393,6 +400,17 @@ func (at *dotnetContainerAppTarget) Deploy(
 		if err != nil {
 			return nil, fmt.Errorf("deploying bicep template: %w", err)
 		}
+		if slices.ContainsFunc(deploymentResult.Resources,
+			func(resource *armresources.ResourceReference) bool {
```

Comment:
Nit: Consider declaring the comparison function outside of the `ContainsFunc()` call to making this a bit easier to read.